modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/outgoing/OutgoingSnapshotCommonTest.java

```diff
@@ -76,12 +64,13 @@ void returnsKeyFromStorage() {
 
     @Test
     void sendsSnapshotMeta() {
-        when(mvPartitionStorage.lastAppliedIndex()).thenReturn(100L);
-        lenient().when(mvPartitionStorage.lastAppliedTerm()).thenReturn(3L);
-        when(txStateStorage.lastAppliedIndex()).thenReturn(100L);
-        lenient().when(txStateStorage.lastAppliedTerm()).thenReturn(3L);
+        lenient().when(partitionAccess.minLastAppliedIndex()).thenReturn(90L);
```

Comment:
These `lenient()`s seem redundant and should be dropped

Reply:
I tried to delete it, but got the following error: `Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.`

Reply:
Then how about removing the whole line? If Mockito says the stubbing is not used, then we probably don't need it at all, right?

Reply:
Fix it